terraform/performance/main.tf

```diff
@@ -109,6 +109,14 @@ resource "null_resource" "validator_linux" {
   }
   provisioner "remote-exec" {
     inline = [
+      #mock server dependencies getting transfered.
+      # todo make this optional with a env var.
+      "git clone https://github.com/aws/amazon-cloudwatch-agent-test.git",
+      "cd amazon-cloudwatch-agent-test && git checkout xray-performance-test",
```

Comment:
I know this was for testing, but for the PR, this should be checking out `main`.